### DISCUSSION TITLE
Generate tags but not release

### DISCUSSION THREAD

**Discussion by bombillazo** [semantic-release/semantic-release#2475] [Q&A]
Which plugin or configuration is needed to create a new tag for the version but **_NOT_** create a release? I have the following:
```
  plugins: [
    '@semantic-release/commit-analyzer',
    '@semantic-release/release-notes-generator',
    '@semantic-release/changelog',
    '@semantic-release/npm',
    [
      '@semantic-release/git',
      {
        assets: ['package.json', 'CHANGELOG.md'],
        message:
          'chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}',
      },
    ],
    [
      '@semantic-release/github',
      {
        assets: [],
        releasedLabels: false,
      },
    ],
  ],
```

There is no option on the `@semantic-release/github` plugin to only create the tag, nor any plugin that only creates tag. :/

**Comment by melink14** [comment]
I think it should work if you just include `commit-analyzer`. It looks like your config is doing a lot more than just making a tag though. Do you also want to push a commit with package.json and CHANGELOG.md?

**Reply by bombillazo** [reply]
I am generating the ChangeLog file with `commit-analyzer` + `release-notes-generator` +  `changelog`, updating the package.json with `npm`, committing the changes to the repo with `git` and using `github` to create the tag _**BUT**_ it is also creating a release. I want to only create a using the `github` plugin but see no option to skip releases.

**Reply by travi** [reply]
tags are handled in core and do not require the github plugin. if you do not want to create a github release, simply do not include the github plugin
